core/ulog/README.md

```diff
@@ -0,0 +1,49 @@
+# Ulog package
+
+ulog provides an interface and abstraction layer over the logger implementation used underneath,
```

Comment:
changed to "ulog provides an API wrapper around the logging library (zap Logger)"